core/src/main/java/hudson/cli/GroovyCommand.java

```diff
@@ -70,7 +72,9 @@ protected int run() throws Exception {
         binding.setProperty("stderr", stderr);
 
         GroovyShell groovy = new GroovyShell(Jenkins.get().getPluginManager().uberClassLoader, binding);
-        groovy.run(loadScript(), "RemoteClass", remaining.toArray(new String[0]));
+        String script = loadScript();
+        ScriptListener.fireScriptEvent(script, "CLI", User.current());
```

Comment:
```suggestion
        ScriptListener.fireScriptEvent(script, "CLI/GroovyCommand", User.current());
```

💡 As you include the origin, I imagine you would like to use it for auditing purpose, so having more information between the two CLI calls could be interesting. WDYT?
(And another one for the GroovyshCommand of course)